modules/update-best-practices.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-best-practices_{context}"]
+= Best practices for cluster updates
+
+{product-title} is designed to provide a robust update experience that allows clusters to update with minimal disruptions to workloads.
+Updates will not begin unless the cluster is determined to be in an upgradeable state at the time of the update request.
+
+While this design helps ensure that updates are successful as long as some key conditions are met, there are a number of actions you can take to increase your chances of a successful cluster update.
+
+[discrete]
+[id="recommended-versions_{context}"]
+=== Choose versions recommended by the OpenShift Update Service
+
+The OpenShift Update Service (OSUS) provides update recommendations based on cluster characteristics such as the cluster's subscribed channel, which are then saved by the Cluster Version Operator as either recommended or conditional updates.
+While it is possible to attempt an update to a version that is not recommended by OSUS, doing so significantly increases the risk of update failure or unintended consequences to the cluster after the update has finished.
```

Comment:
Can we spin this into a positive-sounding (or at least less negative) statement? I think "significantly increases risk" is not even that much true. "update failure" is very strong term, but actually very rare, most upgrade issues are annoyances. I'd say recommended paths are near-zero risk and non-recommended is low risk, we just don't have enough data to actually claim near-zero risk (even if it likely is safe - we do a *lot* of testing, and we do not have enough  resources to test every possible path).

So I would maybe just say that following recommended paths protects the users from encountering known issues or unintended consequences, or something like that.

Reply:
I'll replace my wording with yours, sure thing.